frontend/src/metabase/query_builder/components/dataref/DatabasePane.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import MainPane from "./MainPane";
+
+const DatabasePane = ({ database, ...props }) => (
+  <MainPane {...props} databases={[database]} />
```

Comment:
Did you decide not to do a different pane for a single database vs. all the databases?